plugin/keyring_vault/vault_credentials_parser.cc

```diff
@@ -39,18 +40,18 @@ namespace keyring
       iter->second.clear();
```

Comment:
It is secure. All memory allocated by Secure_string must be deallocated by Secure_string deallocator. Thus in the end when this memory is going to be freed it will be deallocated by secure deallocator which will clean the memory.